components/base/FooterCTA.tsx

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import styled from "@emotion/styled";
+import Button from "./Button";
+import { PURPLE_DARK } from "../colors";
+import { H2, SubHeader } from "./Typography";
+import { maxWidth, BREAKPOINT_IPAD, BREAKPOINT_MOBILE } from "../breakpoints";
+
+// Styles
+
+const Container = styled.div`
+  padding: 60px;
+  width: 100%;
+  background-color: ${PURPLE_DARK};
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  @media (${maxWidth(BREAKPOINT_MOBILE)}) {
+    padding: 10px;
+    flex-direction: column;
+  }
+  /* Text sizes decrease in smaller screens */
+  > div > h2 {
+    @media (${maxWidth(BREAKPOINT_IPAD)}) {
+      font-size: 18px;
+      line-height: 22px;
+    }
+  }
+  > div > span {
+    @media (${maxWidth(BREAKPOINT_IPAD)}) {
+      font-size: 16px;
+      line-height: 20px;
+    }
+  }
+`;
+
+// Props
+type ButtonProps = {
+  text: string;
+};
+type Props = { topic: string; description: string; button?: ButtonProps };
+
+function FooterCTA({ topic, description, button }: Props) {
+  return (
+    <Container>
+      <div>
+        <H2 isDark>{topic}</H2>
```

Comment:
Looks like the margins on this standard h2 are bumping the text off of the vertical center of the component and adding a lot of space between the header and sub-header compared to the designs

Reply:
I ended up unsetting the default margins, and added a bottom margin to the elements. This allows there to be a margin between the bottom text and the button on mobile too.